app/models/item.rb

```diff
@@ -21,4 +24,16 @@ def self.five_popular(var)
           .order("total_orders #{var}")
           .limit(5)
   end
+
+  def self.enabled_items
+    Item.all.where(enabled: true)
```

Comment:
Small, but do you need the ‘Item.all’ here? Should just be able to do ‘where’...

Reply:
You're totally right - good catch! for some reason I was getting stuck thinking I needed to call the where on all. Changing now!